arm/Microsoft.Sql/servers/readme.md

```diff
@@ -18,8 +18,9 @@ This module deploys a SQL server.
 
 | Parameter Name | Type | Default Value | Possible Values | Description |
 | :-- | :-- | :-- | :-- | :-- |
-| `administratorLogin` | string |  |  | Required. Administrator username for the server. |
-| `administratorLoginPassword` | secureString |  |  | Required. The administrator login password. |
+| `administrators` | object |  |  | Conditional. Azure Active Directory admin of the server. |
```

Comment:
Update to latest perameter descriptions. Feel free to use the `Set-ModuleReadMe.ps1` utility for that